core/controllers/admin.py

```diff
@@ -748,11 +748,31 @@ def get(self):
         self.render_json(response)
 
 
-class AddContributionRightsHandler(base.BaseHandler):
-    """Handles adding contribution rights for contributor dashboard page."""
+class ContributionRightsHandler(base.BaseHandler):
+    """Handles contribution rights of a user on contributor dashboard page."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
+    @acl_decorators.can_access_admin_page
+    def get(self):
+        username = self.request.get('username', None)
+        if username is None:
+            raise self.InvalidInputException('Missing username param')
+        user_id = user_services.get_user_id_from_username(username)
+        if user_id is None:
+            raise self.InvalidInputException(
+                'Invalid username: %s' % username)
```

Comment:
Can we extract this logic into a reusable helper function so that we can use it the delete handler as well?

Reply:
Actually this will be removed finally because of the args validation that is being introduced. So once [#13223](https://github.com/oppia/oppia/pull/13223) is merged, I will be removing these.